src/settings.rs

```diff
@@ -159,8 +160,8 @@ fn set_cookies_method(req: Request<Body>, remove_cookies: bool) -> Response<Body
 		.unwrap_or_else(String::new); // Return an empty string if None
 
 	// If there are subscriptions to restore set them and delete any old subscriptions cookies, otherwise delete them all
-	if subscriptions.is_some() {
-		let sub_list: Vec<String> = subscriptions.expect("Subscriptions").split('+').map(str::to_string).collect();
+	if let Some(subscriptions) = subscriptions {
+		let sub_list: Vec<String> = subscriptions.split('+').map(str::to_string).collect();
```

Comment:
You can leave this unpatched (as well as :213-214) - I'd rather atomic PR's and fixed this in `main` - ignore the CI/CD checks